scripts/generate_rust_target.rs

```diff
@@ -154,7 +154,6 @@ fn main() {
         );
         ts.push("dynamic-linking", true);
         ts.push("crt-static-respected", true);
-        ts.push("executables", true);
```

Comment:
Maybe set this to false? This indicates if executables are supported or not. For these targets executables are not supported.

Reply:
It defaults to false, so it should be fine.